src/components/AddCar.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable */
+
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { addCar } from '../redux/actions/CarAction';
+
+const AddCar = () => {
```

Comment:
- [ ] The functionality to delete cars is there because if I add a car, I get the opportunity to delete any other.

However, I can't see any link for deleting cars.  Kindly add a link so you can delete cars without having to add another one first. 👍